Drop after x11-toolkits/libhandy >= 1.8.0 update

src/meson.build:77:2: ERROR: Dependency lookup for libhandy-1 with method 'pkgconfig' failed: Invalid version, need 'libhandy-1' ['>= 1.8.0'] found '1.6.2'.

--- src/meson.build.orig	2024-02-10 16:19:01 UTC
+++ src/meson.build
@@ -74,7 +74,7 @@ app_deps = [
   dependency('gio-unix-2.0', version: '>= 2.50'),
   dependency('gtk+-3.0', version: '>= 3.22'),
   dependency('json-glib-1.0', version: '>= 1.0'),
-  dependency('libhandy-1', version: '>= 1.8.0'),
+  dependency('libhandy-1', version: '>= 1.6.0'),
   dependency('granite', version: '>= 6.2.0'),
   dependency('gtk-layer-shell-0',
     fallback: ['gtk-layer-shell-0', 'gtk-layer-shell'],
